eos-tech-support/eos-write-usb-internet

```diff
@@ -0,0 +1,243 @@
+ROAY70WqlJxDyEbD+ctTr0C20uoRnalc83CNzeKvAQAACQAAoGUAAAAxU1BT7TC92kMAiUen+NATpHNm
+IkkAAABkAAAAAB8AAAAcAAAALgBrAGkAdwBpAHgALQB3AGkAbgBkAG8AdwBzACAAKABIADoAXABjAG8A
+bgB0AGUAbgB0ACkAAAAAAAAAuQAAADFTUFMw8SW370caEKXxAmCMnuusNQAAAAoAAAAAHwAAABIAAABr
+AGkAdwBpAHgALQBkAGUAcwBrAHQAbwBwAC4AZQB4AGUAAAAVAAAADwAAAABAAAAAADYwNLpa1gEVAAAA
+DAAAAAAVAAAA+AlKAAAAAAApAAAABAAAAAAfAAAACwAAAEEAcABsAGkAYwBhAGMAaQDzAG4AAAAAABUA
+AAAOAAAAAEAAAAAAvcM3ulrWAQAAAACFAAAAMVNQU6ZqYyg9ldIRtdYAwE/ZGNBpAAAAHgAAAAAfAAAA
+LAAAAEgAOgBcAGMAbwBuAHQAZQBuAHQAXAAuAGsAaQB3AGkAeAAtAHcAaQBuAGQAbwB3AHMAXABrAGkA
+dwBpAHgALQBkAGUAcwBrAHQAbwBwAC4AZQB4AGUAAAAAAAAAAAAAAAAAAAA=
+BASE64
+)
+
+kolibri_windows_lnk_base64_data=$(cat <<BASE64
+TAAAAAEUAgAAAAAAwAAAAAAAAEbjAggAIAAAAKzpU1dkYNYBFChhV2Rg1gEUKGFXZGDWASiTRgAAAAAA
+AQAAAAAAAAAAAAAAAAAAAOsAFAAfUOBP0CDqOmkQotgIACswMJ0ZAC9DOlwAAAAAAAAAAAAAAAAAAAAA
+AAAAVgAxAAAAAAD7UCmkEABXaW5kb3dzAEAACQAEAO++c06sJPtQKaQuAAAACiMKAAAABQAAAAAAAAAA
+AAAAAAAAANWztQBXAGkAbgBkAG8AdwBzAAAAFgBmADIAKJNGAPZQe6EgAGV4cGxvcmVyLmV4ZQAASgAJ
+AAQA7772UHuh9lB7oS4AAADmvAsAAAAPAAAAAAAIAQAAAAAAAAAA7PhqAGUAeABwAGwAbwByAGUAcgAu
+AGUAeABlAAAAHAAAAEYAAAAcAAAAAQAAABwAAAAtAAAAAAAAAEUAAAARAAAAAwAAAJhEo1YQAAAAAEM6
+XFdpbmRvd3NcZXhwbG9yZXIuZXhlAAAeACIALgBrAG8AbABpAGIAcgBpAC0AdwBpAG4AZABvAHcAcwBc
+AEsAbwBsAGkAYgByAGkALgBlAHgAZQAiAD8ASAA6AFwAYwBvAG4AdABlAG4AdABcAC4AawBvAGwAaQBi
+AHIAaQAtAHcAaQBuAGQAbwB3AHMAXABrAG8AbABpAGIAcgBpAFwAYwBvAHIAZQBcAHMAdABhAHQAaQBj
+AFwAYQBzAHMAZQB0AHMAXABsAG8AZwBvAC4AaQBjAG8AFAMAAAEAAKAld2luZGlyJVxleHBsb3Jlci5l
+eGUAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+ACUAdwBpAG4AZABpAHIAJQBcAGUAeABwAGwAbwByAGUAcgAuAGUAeABlAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQAAAABQAAoCQAAACDAAAAHAAA
+AAsAAKAE9IvzQx3yQpMFZ94LKPwjgwAAAGAAAAADAACgWAAAAAAAAABlb3ItZGVza3RvcAAAAAAAZCqr
+ROAY70WqlJxDyEbD+ctTr0C20uoRnalc83CNzeJkKqtE4BjvRaqUnEPIRsP5y1OvQLbS6hGdqVzzcI3N
+4gAAAAA=
+BASE64
+)
+
+kiwix_library_xml=$(cat <<XML
+<library version="20110515">
+    <book
+        id="97e3f348-38f4-6787-b95d-1e4886bd803c"
+        path="../.kiwix/files/com.endlessm.encyclopedia.en-openzim-subscription/1.zim"
```

Comment:
This will need to be in synch with what's expected in Endless OS / Flatpak land, c.f. https://github.com/endlessm/eos-boot-helper/pull/305